ext/pdo_mysql/mysql_driver.c

```diff
@@ -459,7 +459,19 @@ static bool pdo_mysql_set_attribute(pdo_dbh_t *dbh, zend_long attr, zval *val)
 			((pdo_mysql_db_handle *)dbh->driver_data)->fetch_table_names = bval;
 			PDO_DBG_RETURN(true);
 
-#ifndef PDO_USE_MYSQLND
+#ifdef PDO_USE_MYSQLND
+		case PDO_ATTR_STRINGIFY_FETCHES:
+			if (!pdo_get_bool_param(&bval, val)) {
+				return false;
```

Comment:
```suggestion
				PDO_DBG_RETURN(false);
```

Reply:
Which... while you here might want to fix the other cases which for some reason just do ``return false;`` ...